client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriverAuth.java

```diff
@@ -410,7 +410,7 @@ public void testFailedNoneSslVerificationWithSSL()
     {
         assertThatThrownBy(() -> createConnection(ImmutableMap.of("SSLVerification", "NONE")))
                 .isInstanceOf(SQLException.class)
-                .hasMessage("Connection property SSLTrustStorePath cannot be set if SSLVerification is set to NONE");
+                .hasMessageContaining("Connection property SSLTrustStorePath cannot be set if SSLVerification is set to NONE");
```

Comment:
What is the full message now? Error messages of JDBC is an UX change, is it possible to avoid such changes?

Reply:
All errors are reported, not a first that was encountered